### PR TITLE
dankness over 9000

### PR DESCRIPTION
**Pull request by vikRrrr**
- Use [Standard](https://github.com/feross/Standard) style and change all JS accordingly
- Use [axios](https://github.com/mzabriskie/axios) for all HTTP requests and clear out other deps used for that (I found `request`, `request-promise` and `unirest` when doing that)
- Make the `xkcd` command send an embed
- Moved assets (`arrays.json`, `red.png`, `triggered.jpg`) into their own directory
- Some other misc. fixes/minor changes

No features were added or removed, nor changed in a major way.
Basically just consistency and code quality changes. And more ES6 ~~dank~~goodness.

`var` is now illegal

### CODE REVIEW

**Comment by vikRrrr** [comment]
Damn, I commit way too much.

File: commands/bug.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
         msg.react('👌')
-        client.users.get(config.owner).sendMessage('**' + msg.author.username + '#' + msg.author.discriminator + ' (' + msg.author.id + ')' + ':**\n' + args.join(' '))
-        msg.reply('Thanks for bugging Melmsie with a bug request!')
+        const { username, discriminator, id } = msg.author
+        const report = args.join(' ')
+
+        client.users.get(config.owner)
+          .sendMessage(`**${username}#${discriminator} (${id}):** ${report}`)
+
+        msg.reply('thanks for bugging Melmsie with a bug request!')
```
Comment (melmsie) [inline commands/bug.js]: Why make everything lowercase?

Comment (vikRrrr) [inline commands/bug.js]: Just because since it is using Message#reply and it will show as `@mention, you need to...` in the end, I thought it made more sense to do so.

**Comment by melmsie** [comment]
👌
